### PR TITLE
Add release notes for v1.7

### PR DESCRIPTION
**Pull request by aleks-p**

### CODE REVIEW

**Review by simonswine — APPROVED** [review]
LGTM

File: docs/sources/release-notes/v1-7.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Version 1.7 release notes
+
+We are excited to present Grafana Pyroscope 1.7.
+
+This release includes several new features such as:
+
+* The ability to relabel profiles at ingest time
+* Per-app (service) usage metrics
+* Stacktrace selectors for merge profile queries
+* profile `pprof` export tailored to Go PGO
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:17]:
```suggestion
This release includes several new features:

* The ability to relabel profiles at ingest time
* Per-app (service) usage metrics
* Stacktrace selectors for merge profile queries
* Profile `pprof` export tailored to Go PGO
```

File: docs/sources/release-notes/v1-7.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Additionally, numerous other changes improve stability, performance, and documentation.
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:19]:
```suggestion
Additionally, we've improved stability, performance, and documentation.
```

File: docs/sources/release-notes/v1-7.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Notable changes are listed below. For more details, check out the Full 1.7.0 Changelog: https://github.com/grafana/pyroscope/compare/v1.6.0...v1.7.0.
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:21]:
```suggestion
Notable changes are listed below. For more details, check out the full 1.7.0 changelog: https://github.com/grafana/pyroscope/compare/v1.6.0...v1.7.0.
```

File: docs/sources/release-notes/v1-7.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Improvements and Updates
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:23]: Changing heading level.
```suggestion
## Improvements and updates
```

File: docs/sources/release-notes/v1-7.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+* `pprof` export for Go PGO ([#3360](https://github.com/grafana/pyroscope/pull/3360))
+* Custom binary format for symdb ([#3138](https://github.com/grafana/pyroscope/pull/3138))
+* Repair truncated Go CPU profiles ([#3344](https://github.com/grafana/pyroscope/pull/3344))
+* Add initial load tests ([#3331](https://github.com/grafana/pyroscope/pull/3331))
+* Align default step for `/render` with Grafana ([#3326](https://github.com/grafana/pyroscope/pull/3326))
+* Allow use of different protocols in `profilecli` ([#3368](https://github.com/grafana/pyroscope/pull/3368))
+* Various performance improvements (#3395, #3345, #3349, #3351, #3386, #3348, #3358)
+* Improve readiness check for ingesters and frontend ([#3435](https://github.com/grafana/pyroscope/pull/3435))
+
+### Fixes
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:40]:
```suggestion
## Fixes
```

File: docs/sources/release-notes/v1-7.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Version 1.7 includes the following fixes:
+
+* Fix error handling in filterProfiles ([#3338](https://github.com/grafana/pyroscope/pull/3338))
+* Fix frontend header handling ([#3363](https://github.com/grafana/pyroscope/pull/3363))
+* Fix line numbers for pyspy ([#3337](https://github.com/grafana/pyroscope/pull/3337))
+* Don't compute delta on relabeled `godeltaprof` memory profiles ([#3398](https://github.com/grafana/pyroscope/pull/3398))
+* Honor stacktrace partitions at downsampling ([#3408](https://github.com/grafana/pyroscope/pull/3408))
+* Fix infinite loop in index writer ([#3356](https://github.com/grafana/pyroscope/pull/3356))
+
+### Documentation Improvements
```
Comment (knylander-grafana) [inline docs/sources/release-notes/v1-7.md:51]:
```suggestion
### Documentation improvements
```

**Review by knylander-grafana — APPROVED** [review]
Minor changes for heading levels and wording. Thank you for adding the release notes!

**Pull request merged**